litestar/contrib/sqlalchemy/alembic/commands.py

```diff
@@ -0,0 +1,255 @@
+    return alembic_cfg
+
+
+def upgrade(
+    app: Litestar,
+    revision: str = "head",
+    sql: bool = False,
+    tag: str | None = None,
+) -> None:
+    """Create or upgrade a database."""
+    plugin = app.plugins.get(SQLAlchemyInitPlugin)
+    alembic_cfg = get_alembic_command_config(config=plugin._config)
+    migration_command.upgrade(config=alembic_cfg, revision=revision, tag=tag, sql=sql)
+
+
+def downgrade(
+    app: Litestar,
+    revision: str = "head",
+    sql: bool = False,
+    tag: str | None = None,
+) -> None:
+    """Downgrade a database to a specific revision."""
+    plugin = app.plugins.get(SQLAlchemyInitPlugin)
+    alembic_cfg = get_alembic_command_config(config=plugin._config)
+    migration_command.downgrade(config=alembic_cfg, revision=revision, tag=tag, sql=sql)
+
+
+def check(
+    app: Litestar,
+) -> None:
+    """Check if revision command with autogenerate has pending upgrade ops."""
+    plugin = app.plugins.get(SQLAlchemyInitPlugin)
+    alembic_cfg = get_alembic_command_config(config=plugin._config)
+    migration_command.check(config=alembic_cfg)
+
+
+def current(app: Litestar, verbose: bool = False) -> None:
+    """Display the current revision for a database."""
+    plugin = app.plugins.get(SQLAlchemyInitPlugin)
+    alembic_cfg = get_alembic_command_config(config=plugin._config)
+    migration_command.current(alembic_cfg, verbose=verbose)
+
+
+def edit(app: Litestar, revision: str) -> None:
+    """Edit revision script(s) using $EDITOR."""
+    plugin = app.plugins.get(SQLAlchemyInitPlugin)
+    alembic_cfg = get_alembic_command_config(config=plugin._config)
+    migration_command.edit(config=alembic_cfg, rev=revision)
+
+
+def ensure_version(app: Litestar, sql: bool = False) -> None:
+    """Create the alembic version table if it doesn't exist already."""
+    plugin = app.plugins.get(SQLAlchemyInitPlugin)
+    alembic_cfg = get_alembic_command_config(config=plugin._config)
+    migration_command.ensure_version(config=alembic_cfg, sql=sql)
+
+
+def heads(app: Litestar, verbose: bool = False, resolve_dependencies: bool = False) -> None:
+    """Show current available heads in the script directory."""
+    plugin = app.plugins.get(SQLAlchemyInitPlugin)
+    alembic_cfg = get_alembic_command_config(config=plugin._config)
+    migration_command.heads(config=alembic_cfg, verbose=verbose, resolve_dependencies=resolve_dependencies)  # type: ignore[no-untyped-call]
+
+
+def history(
+    app: Litestar,
+    rev_range: str | None = None,
+    verbose: bool = False,
+    indicate_current: bool = False,
+) -> None:
+    """List changeset scripts in chronological order."""
+    plugin = app.plugins.get(SQLAlchemyInitPlugin)
+    alembic_cfg = get_alembic_command_config(config=plugin._config)
+    migration_command.history(
+        config=alembic_cfg, rev_range=rev_range, verbose=verbose, indicate_current=indicate_current
+    )
+
+
+def merge(
+    app: Litestar,
+    revisions: str,
```

Comment:
Is this correct? Shouldn't this be `tuple[str, str]` or something like that?

Reply:
I'll take a look at this one.  I started by making wrappers around each alembic command and I'm slowly adding all the commands to the CLI.